### PR TITLE
Add a README.md to apprtc directory.

### PR DESCRIPTION
**Pull request by milkypostman**
The README initially contains how to run AppRTC locally.

This is where I will add more information eventually which is needed when developing locally.

@jiayliu

### CODE REVIEW

File: samples/web/content/apprtc/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# AppRTC Demo Code
+
+## Local dev server
+
+Development AppRTC server can be accessed by visiting [http://localhost:8080](http://localhost:8080).
+
+### Prerequisites
+
+Running AppRTC localling requires the [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python).
```
Comment (samdutton) [inline samples/web/content/apprtc/README.md]: locally

Comment (milkypostman) [inline samples/web/content/apprtc/README.md]: done

**Comment by jiayliu** [comment]
This should include some instruction on grunt, which is required for local development.

**Comment by samdutton** [comment]
Could link to the Developer's Guide grunt section
<https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit#heading=h.ecwmnvuup8pc>
.

**Comment by milkypostman** [comment]
PTAL. I would ask that we get this in and then work on revisions as documentation should be low overhead.

File: samples/web/content/apprtc/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            grunt-contrib-csslint \
+            grunt-htmlhint \
+            grunt-jscs \
+            grunt-contrib-jshint \
+            grunt-shell \
+            grunt-jstestdriver-phantomjs \
+            grunt-closurecompiler \
+            grunt-contrib-compress \
+            grunt-jinja
+```
```
Comment (chuckhays) [inline samples/web/content/apprtc/README.md]: I think this can just be "npm install" in the dir with packages.json.

**Comment by chuckhays** [comment]
LGTM, I think we can simplify the npm install line, but we can revise it like you said.

File: samples/web/content/apprtc/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+The development AppRTC server can be accessed by visiting [http://localhost:8080](http://localhost:8080).
+
+Running AppRTC locally requires the [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python) and [Grunt](http://gruntjs.com/).
+
+Detailed instructions for running on Ubuntu Linux are provided below.
+
+### Running on Ubuntu Linux
+
+Install grunt by first installing [npm](https://www.npmjs.com/),
```
Comment (jiayliu) [inline samples/web/content/apprtc/README.md]: I think all the grunt installing related stuff can be replaced by a link to http://gruntjs.com/getting-started.

File: samples/web/content/apprtc/README.md (hunk 2 above)
Comment (milkypostman) [inline samples/web/content/apprtc/README.md]: done

File: samples/web/content/apprtc/README.md (hunk 3 above)
Comment (milkypostman) [inline samples/web/content/apprtc/README.md]: ok, added a reference to it below.

**Comment by milkypostman** [comment]
Tested and it looks like `npm install` will work from any child directory as well. and seems to respect the root of the project. pretty interesting.

**Pull request merged**